src/h3lib/include/h3api.h.in

```diff
@@ -292,79 +298,79 @@ double H3_EXPORT(cellAreaKm2)(H3Index h);
 double H3_EXPORT(cellAreaM2)(H3Index h);
 /** @} */
 
-/** @defgroup edgeLength edgeLength
- * Functions for edgeLength
+/** @defgroup getHexagonEdgeLengthAvg getHexagonEdgeLengthAvg
+ * Functions for getHexagonEdgeLengthAvg
  * @{
  */
 /** @brief average hexagon edge length in kilometers (excludes pentagons) */
-double H3_EXPORT(edgeLengthKm)(int res);
+double H3_EXPORT(getHexagonEdgeLengthAvgKm)(int res);
 
 /** @brief average hexagon edge length in meters (excludes pentagons) */
-double H3_EXPORT(edgeLengthM)(int res);
+double H3_EXPORT(getHexagonEdgeLengthAvgM)(int res);
```

Comment:
Actually, we should just call this `getEdgeLengthAvg`, since I'll be computing the new average edge length values over **all** edges, without excluding any that might be incident with a pentagon.